os/kernel/wdog/wd_start.c

```diff
@@ -494,6 +470,21 @@ void wd_timer(void)
 #ifdef CONFIG_SCHED_TICKSUPPRESS
 void wd_timer_nohz(clock_t ticks)
 {
-	g_wd_ticks += ticks;
+	FAR struct wdog_s *wdog;
+	int decr;
+
+	for (wdog = g_wdactivelist.head; ticks > 0 && wdog; wdog = wdog->next) {
+
+		/* Decrement the lag for this watchdog. */
+
+		decr = MIN(wdog->lag, ticks);
+
+		/* There are.  Decrement the lag counter */
+
+		wdog->lag -= decr;
+		ticks -= decr;
```

Comment:
Since it has been modify from g_wd_ticks to ticks, I think we can remove this line?

Reply:
Other than this I have no comment on the PR.

Reply:
The decr can be `wdog->lag`, not `ticks`. This case, we need this line.
When the decr is `ticks`, this line is unnecessary.

Reply:
Did I get your point exactly?

Reply:
yes, correct